extensions/common/iam/oauth2/oauth2-daps/src/test/java/org/eclipse/edc/iam/oauth2/daps/Rfc8707IntegrationTest.java

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
This header needs to be changed to your corporate sponsor that aligns with the CLA

Reply:
In progress on our side